OneCommunityGlobal/HighestGoodNetworkApp
# Ramya teams limit see all

# Description of the feature

- A “Limit See-All” option for teams 

-  Other Links ->Teams

- When this option is turned on for a team member,  the team member  shouldn't  be able to see  other members of the team in their dashboard 

- The member however should be able to see team members of other teams  they are part of 



## Related PRS (if any):
This frontend PR is related to the [576](https://github.com/OneCommunityGlobal/HGNRest/pull/576) backend PR.

## Main changes explained:
-  Added a column ‘See all’ in the "Team Menber " popup
-  A tool tip beside See All
-  Added a Toggle Switch in Add/Edit Teams pop-up beside every row 
-  Renderd the toggle switch with apprpriate choice.
-  When the user makes the choice,  make a call to   action function 
-  The  action fiunction  makes a post call to the backend 
-  Make a dispatch call to the reducer to update the redux store. 

## How to test:
1.  check into current branch
2.  do `npm install` and `...` to run this PR locally
3.  log as admin user
4.  Go to Other Links ->Teams
5.  Create two Teams , lets say Team A and Team B (you can use any existing Team as well)
6.  Add  the same  team member to Team A and Team B
7.  Enable the toggle switch in Team A for this member
8.  Login to the user's profile and verify if the user is only able  to see  the team members of the  team B and not Team A

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/89441435/2eaaf5a2-9f80-442c-9ace-20fddc997f35


## Note:

- Kindly mention any  bugs , errors you see in the console. 
- Try breaking the functionality , any suggestion is helpful
